screens/SettingsScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import { StatusBar } from 'expo-status-bar'
+import { useEffect, useState } from 'react'
+import { Alert, Button, Platform, StyleSheet, TextInput } from 'react-native'
+
+import { View } from '../components/Themed'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1
+  },
+  title: {
+    fontSize: 20,
+    fontWeight: 'bold'
+  },
+  separator: {
+    marginVertical: 30,
+    height: 1,
+    width: '80%'
+  }
+})
+
+export default function SettingsScreen () {
+  const [serverUrl, setServerUrl] = useState('')
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+
+  async function getSettings () {
+    try {
+      const _serverUrl = await AsyncStorage.getItem('serverUrl')
+      if (_serverUrl !== null) {
+        setServerUrl(_serverUrl)
+      }
```

Comment:
suggestion: just in case user save empty string, same for others

```suggestion
      if (!_serverUrl) {
        setServerUrl(_serverUrl)
      }
```